rest-api-spec/src/main/resources/rest-api-spec/api/ml.stop_datafeed.json

```diff
@@ -32,12 +32,6 @@
         "required":false,
         "description":"Whether to ignore if a wildcard expression matches no datafeeds. (This includes `_all` string or when no datafeeds have been specified)"
       },
-      "allow_no_datafeeds":{
-        "type":"boolean",
-        "required":false,
-        "description":"Whether to ignore if a wildcard expression matches no datafeeds. (This includes `_all` string or when no datafeeds have been specified)",
-        "deprecated":true
-      },
```

Comment:
Why is this deleted? Seems to be unrelated to the PR intent.

Reply:
It was deprecated in #60601 and removed later but this bit of client spec was deleted when the code handling the `allow_no_datafeeds` parameter was deleted